src/seedu/addressbook/Main.java

```diff
@@ -106,6 +106,9 @@ private void recordResult(CommandResult result) {
     private CommandResult executeCommand(Command command)  {
         try {
             command.setData(addressBook, lastShownList);
+            if(!storage.fileExists()) {
+            	ui.showFileDoesNotExistMessage(storage.getPath());
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.